<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1225</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Recorder</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1211</width>
     <height>621</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="12,16">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="2,1">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item>
       <widget class="QGraphicsView" name="color_image"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1,1,0,2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,6">
            <item>
             <widget class="QLabel" name="width_label">
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);
font: 25 9pt &quot;Noto Sans JP Light&quot;;</string>
              </property>
              <property name="text">
               <string>cutoff    </string>
              </property>
              <property name="margin">
               <number>4</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="cutoff_slider">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>150</number>
              </property>
              <property name="value">
               <number>150</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="2,6">
            <item>
             <widget class="QLabel" name="width_label_2">
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);
font: 25 9pt &quot;Noto Sans JP Light&quot;;</string>
              </property>
              <property name="text">
               <string>variance</string>
              </property>
              <property name="margin">
               <number>4</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="variance_slider">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>200</number>
              </property>
              <property name="value">
               <number>100</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QCheckBox" name="checkBox_1">
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);
font: 63 9pt &quot;Noto Sans JP SemiBold&quot;;</string>
              </property>
              <property name="text">
               <string>1</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_2">
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);
font: 63 9pt &quot;Noto Sans JP SemiBold&quot;;</string>
              </property>
              <property name="text">
               <string>2</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_3">
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);
font: 63 9pt &quot;Noto Sans JP SemiBold&quot;;</string>
              </property>
              <property name="text">
               <string>3</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_4">
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);
font: 63 9pt &quot;Noto Sans JP SemiBold&quot;;</string>
              </property>
              <property name="text">
               <string>4</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_5">
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);
font: 63 9pt &quot;Noto Sans JP SemiBold&quot;;</string>
              </property>
              <property name="text">
               <string>5</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QRadioButton" name="radioButton_Nolinear">
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);
font: 63 9pt &quot;Noto Sans JP SemiBold&quot;;</string>
              </property>
              <property name="text">
               <string>nolinear</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_Linear">
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);
font: 63 9pt &quot;Noto Sans JP SemiBold&quot;;</string>
              </property>
              <property name="text">
               <string>Linear</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_Step">
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);
font: 63 9pt &quot;Noto Sans JP SemiBold&quot;;</string>
              </property>
              <property name="text">
               <string>Step</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup</string>
              </attribute>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="Info">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
font: 25 12pt &quot;Noto Sans JP Light&quot;;</string>
            </property>
            <property name="text">
             <string>aaa</string>
            </property>
            <property name="margin">
             <number>5</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGraphicsView" name="depth_map"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,0">
      <item>
       <widget class="GraphicsLayoutWidget" name="graph1_1"/>
      </item>
      <item>
       <widget class="GraphicsLayoutWidget" name="graph1_2"/>
      </item>
      <item>
       <widget class="GraphicsLayoutWidget" name="graph1_3"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsLayoutWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
